spring-cloud-skipper-server-core/src/test/resources/repositories/sources/test/python/python-printer-1.0.0/values.yml

```diff
@@ -0,0 +1,5 @@
+# Default values for {{name}}
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates
+spec:
+  resource: https://github.com/ilayaperumalg/sandbox/raw/master/python/hello.py.zip
```

Comment:
Can we stop using your git repo for these?

Reply:
yeah, this is just a placeholder for the non-java applications. What do you think as a better alternative?

Reply:
Either in repo itself or just attaching files to a dummy gh issue(in other projects I've just created a dummy issue, close it and then you can just drag and drop files there and github creates a link).